main.go

```diff
@@ -103,11 +103,10 @@ func initFlags(fs *pflag.FlagSet) {
 		10*time.Minute,
 		"The minimum interval at which watched resources are reconciled.",
 	)
-
 	fs.StringVar(
 		&options.ProviderIDFormat,
 		"provider-id-fmt",
-		string(options.ProviderIDFormatV1),
+		string(options.ProviderIDFormatV2),
```

Comment:
Since we are making v2 by default, Lets also update the doc to not to explicitly set the provider id format to v2 for example here https://cluster-api-ibmcloud.sigs.k8s.io/topics/powervs/external-cloud-provider.

Also lets check in the whole doc or wherever to update the same. Like manager.yaml https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/config/manager/manager.yaml#L30

Reply:
Updated the docs accordingly.